### PR TITLE
Update README and metadata.json for 7.1.0-preview.7

### PR DESCRIPTION
**Pull request by rjmholt**
# PR Summary

Updates links in the README and the metadata.json file for preview.7.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
I have no idea why the CI tests failed.

File: README.md [hunk 1]
```diff
@@ -87,23 +87,23 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
...
+[pv-windows-64]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/PowerShell-7.1.0-preview.7-win-x64.msi
+[pv-windows-86]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/PowerShell-7.1.0-preview.7-win-x86.msi
+[pv-ubuntu18]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/powershell-preview_7.1.0-preview.7-1.ubuntu.18.04_amd64.deb
+[pv-ubuntu16]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/powershell-preview_7.1.0-preview.7-1.ubuntu.16.04_amd64.deb
+[pv-debian9]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/powershell-preview_7.1.0-preview.7-1.debian.9_amd64.deb
+[pv-debian10]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/powershell-preview_7.1.0-preview.7-1.debian.10_amd64.deb
+[pv-centos]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/powershell-preview-7.1.0_preview.6-1.rhel.7.x86_64.rpm
+[pv-centos8]: https://github.com/PowerShell/PowerShell/releases/download/v7.1.0-preview.7/powershell-preview-7.1.0_preview.6-1.centos.8.x86_64.rpm
```
Comment (TravisEz13) [inline README.md:97]: missed the second version

File: tools/metadata.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "StableReleaseTag": "v7.0.3",
-  "PreviewReleaseTag": "v7.1.0-preview.6",
+  "PreviewReleaseTag": "v7.1.0-preview.7",
   "ServicingReleaseTag": "v6.2.7",
   "ReleaseTag": "v7.0.3",
   "LTSReleaseTag" : ["v7.0.3"],
-  "NextReleaseTag": "v7.1.0-preview.7",
+  "NextReleaseTag": "v7.1.0-rc.1",
```
Comment (TravisEz13) [inline tools/metadata.json:7]: Keep this is a preview, even if it won't be
```suggestion
  "NextReleaseTag": "v7.1.0-preview.8",
```

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

**Comment by rjmholt** [comment]
CI is now passing

**Comment by daxian-dbw** [comment]
Magic! 😄

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**